docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -73,13 +66,11 @@ You need the following to get invited to TechPass and complete the onboarding:
 
    ![proceed-to-mfa-setup](assets/support/more-info-required.png ':size=500')
 
-  </details>
-
+  
 ## Step 3. Configure and verify MFA for TechPass account
 
-<details> <summary style="font-size:20px;font-weight:bold">Set up security verification for TechPass account</summary>
+  ?> This document guides you to configure Microsoft authenticator as your MFA. We recommend Microsoft authenticator for the following reasons:<br>- It supports **Number Matching** to protect you from MFA Fatigue attacks and increases the security of your account.<br>- Microsoft constantly improves its MFA security policies to protect its users.
```

Comment:
Can we have the MFA spelt out in full in the first mention with the abbreviation in parentheses?

Microsoft authenticator vs Microsoft Authenticator?

Since we are trying to help users onboard quickly. Is the info on "number matching" necessary?

Reply:
1. MFA is now spelt out with an abbreviation within the parentheses.
2. Thanks for spotting this @eu-tee  - it should be the latter. fixed.
3. Users can use other Authenticators too, but we recommend Microsoft Authenticator for the mentioned reasons. It will help the users to make an informed decision. Open to discussion, pls let me know if you still think it can be removed. Thanks!